rebench/model/benchmark.py

```diff
@@ -107,3 +107,6 @@ def as_dict(self):
     def from_str_list(cls, data_store, str_list):
         return data_store.get_config(str_list[0], str_list[1], str_list[2],
                                      None if str_list[3] == '' else str_list[3])
+    @classmethod 
+    def get_column_headers(cls):
+        return ["benchmark", "executor", "suite"]
```

Comment:
Could it be that the `extraArgs` column is missing here?
It's included in `as_str_list()` above.